pkg/queue/sharedmain/main.go

```diff
@@ -246,29 +239,39 @@ func Main(opts ...Option) error {
 	// Enable TLS when certificate is mounted.
 	tlsEnabled := exists(logger, certPath) && exists(logger, keyPath)
 
-	mainServer, drainer := buildServer(d.Ctx, env, d.Transport, probe, stats, logger, concurrencyendpoint, false)
+	servingHandler, drainer := servingHandler(d.Ctx, env, d.Transport, probe, stats, logger, concurrencyendpoint)
```

Comment:
`Serving` here and in `servers.go` could be a bit confusing (with Serving as a repo, adminServer is also "serving" stuff). But I don't have a better name ATM.

Reply:
changed to `mainServer` and `mainHandler`